### ISSUE TITLE
Show all custom registration fields

### ISSUE THREAD

**Issue by kewisch** [indico/indico-checkin#28]
We have a "T-Shirt" field in the registration. We'd like to be able to see this when checking someone in. It does not appear there is a way to do this in the Indico check-in app.

Would be great to show all the fields, or a configurable set.

**Comment by ThiefMaster** [comment]
I think the suggestion is reasonable and useful - however, I think the best way of getting this in the app would be someone external contributing it.

Maintainability issues of this rather old app aside (we'd much rather have a more modern one based on React-Native), we currently do not have the resources to make significant changes to the app beyond maintenance (fixing it if broken / adapting to changes in the Indico core).

**Comment by kewisch** [comment]
I suspected as much, and we might indeed contribute this until our next event. Off topic, but have you considered making the QR code just an URL on indico so it could work completely without an app?

**Comment by ThiefMaster** [comment]
I see two problems with using just an URL there:

- less convenient for the person doing the checkin
- impossible to add extra metadata (for example at CERN we also include some extra metadata when people are granted access to the CERN site)

**Comment by ThiefMaster** [comment]
> I suspected as much, and we might indeed contribute this until our next event.

If you are willing to contribute something in this direction, I think it'd be worth to have a chat at some point (before starting development) to sync our ideas on how to best add such a feature.

**Comment by kewisch** [comment]
Absolutely, will get in touch. We're going to do a retro after the event and see which features we need most, and will then reach out.
